### PR TITLE
Add alculquicondor to test/integration/framework OWNERS

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: test/integration/framework/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- alculquicondor
```
Comment (aojea) [inline test/integration/framework/OWNERS:4]:
```suggestion
- alculquicondor #sig-scheduling
- aojea
- wojtek-t
```

File: test/integration/framework/OWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+reviewers:
+- alculquicondor
```
Comment (aojea) [inline test/integration/framework/OWNERS:6]: same https://github.com/kubernetes/kubernetes/pull/113679/files#r1015574290

File: test/integration/framework/OWNERS (hunk 1 above)
Comment (alculquicondor) [reply test/integration/framework/OWNERS:4]: aren't those inherited?

**Comment by aojea** [comment]
/lgtm
/approve

Thanks

File: test/integration/framework/OWNERS (hunk 1 above)
Comment (aojea) [reply test/integration/framework/OWNERS]: yeah, multitasking is not good

**Pull request merged**